babel/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -1039,10 +1039,24 @@ SqlNode LiteralRowConstructor() :
 }
 {
     <LPAREN>
-    e = AtomicRowExpression() { valueList.add(e); }
     (
-        LOOKAHEAD(2)
-        <COMMA> e = AtomicRowExpression() { valueList.add(e); }
+        e = AtomicRowExpression()
```

Comment:
nit: Would it be worth breaking out this parsing logic into a helper function that's called in the initial case as well as in the repeated case below?

Reply:
Yeah I think it'd be better to break it down like that